webpack.config.babel.js

```diff
@@ -121,17 +121,6 @@ export default {
 					name: '[name].[ext]?[hash]'
 				}
 			},
-            {
```

Comment:
Removed this is it was a failed attempt to automatically generate documentation from the js.